### PR TITLE
[cert-manager] docs: secure cert-manager issuer credentials

### PR DESCRIPTION
**Pull request by yalosev**
Signed-off-by: Yuriy Losev <yuriy.losev@flant.com>

## Description
Update documentation

## Why do we need it, and what problem does it solve?
Show the way to a secure ClusterIssuer / Issuer

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests are passed.
- [x] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: cert-manager
type: chore
summary: Add documentation about secure ClusterIssuer to the FAQ
impact_level: low
```

### CODE REVIEW

File: modules/101-cert-manager/docs/FAQ.md [hunk 1]
```diff
@@ -176,3 +176,67 @@ After configuring PKI and enabling Kubernetes [authorization](../../modules/140-
     - www.example.com
   EOF
   ```
+
+## How to secure cert-manager credentials
```
Comment (z9r5) [inline modules/101-cert-manager/docs/FAQ.md:180]:
```suggestion
## How to secure cert-manager credentials?
```

File: modules/101-cert-manager/docs/FAQ.md [hunk 2]
```diff
@@ -176,3 +176,67 @@ After configuring PKI and enabling Kubernetes [authorization](../../modules/140-
...
+
+If you don't want to store credentials in the Deckhouse configmap (security reasons for example), feel free to create
+your own ClusterIssuer / Issuer.
+For example, you can create your own ClusterIssuer for route53 in this way:
```
Comment (z9r5) [inline modules/101-cert-manager/docs/FAQ.md:184]:
```suggestion
If you don't want to store credentials in the `deckhouse` ConfigMap (security reasons for example), feel free to create
your own ClusterIssuer / Issuer.
For example, you can create your own ClusterIssuer for route53 in this way:
```

File: modules/101-cert-manager/docs/FAQ_RU.md [hunk 3]
```diff
@@ -179,3 +179,67 @@ CAA record does not match issuer
     - www.example.com
   EOF
   ```
+
+## Как защитить учетные данные cert-manager
+
+Если вы не хотите хранить учетные данные в Deckhouse ConfigMap (например, по соображениям безопасности), вы можете создать
+свой собственный ClusterIssuer / Issuer.
+Например, вы можете создать свой ClusterIssuer для route53 следующим образом:
```
Comment (z9r5) [inline modules/101-cert-manager/docs/FAQ_RU.md:187]:
```suggestion
Если вы не хотите хранить учетные данные в Deckhouse ConfigMap (например, по соображениям безопасности), вы можете создать
свой собственный ClusterIssuer / Issuer.
Например, вы можете создать свой ClusterIssuer для сервиса [route53](https://aws.amazon.com/route53/) следующим образом:
```

File: modules/101-cert-manager/docs/FAQ.md [hunk 4]
```diff
@@ -176,3 +176,67 @@ After configuring PKI and enabling Kubernetes [authorization](../../modules/140-
...
+1. Create secret with credentials:
```
Comment (z9r5) [inline modules/101-cert-manager/docs/FAQ.md:185]:
```suggestion
1. Create a Secret with credentials:
```

File: modules/101-cert-manager/docs/FAQ_RU.md [hunk 5]
```diff
@@ -179,3 +179,67 @@ CAA record does not match issuer
     - www.example.com
   EOF
   ```
+
+## Как защитить учетные данные cert-manager
```
Comment (z9r5) [inline modules/101-cert-manager/docs/FAQ_RU.md:183]:
```suggestion
## Как защитить учетные данные cert-manager?
```

File: modules/101-cert-manager/docs/FAQ.md (hunk 4 above)
Comment (z9r5) [reply modules/101-cert-manager/docs/FAQ.md:185]: Also, please set indent of code blocks to 3 or there will be a space on the left after rendering.

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @yalosev. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Review by z9r5 — APPROVED** [review]

**Comment by z9r5** [comment]
Cherry-picked into release-1.33

**Pull request merged**